src/main/java/interfaces/Games.java

```diff
@@ -0,0 +1,21 @@
+package interfaces;
+
+import models.GameTypes;
+import models.User;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+public interface Games {
```

Comment:
it seems that we have violations of SOLID principle here.
https://en.wikipedia.org/wiki/SOLID

All these methods are quite different. So we have violations of single responsibility

Reply:
- [x] getters and setters added into AbstractGame class, and deleted in Games interface, now Games has the one method action, choice signature all classes from Games to AbstractGame